databricks/koalas/tests/test_series_plot.py

```diff
@@ -244,11 +244,10 @@ def test_box_summary(self):
         self.assert_eq(expected_whiskers[1], whiskers[1])
         self.assert_eq(expected_fliers, fliers)
 
-    def test_missing(self):
-        ks = self.kdf1['a']
-
-        unsupported_functions = ['kde']
-        for name in unsupported_functions:
-            with self.assertRaisesRegex(PandasNotImplementedError,
-                                        "method.*Series.*{}.*not implemented".format(name)):
-                getattr(ks.plot, name)()
+    def test_kde_plot(self):
+        pdf = self.pdf1
+        kdf = self.kdf1
+
+        ax1 = pdf['a'].plot("kde", colormap='Paired')
+        ax2 = kdf['a'].plot("kde", colormap='Paired')
+        self.compare_plots(ax1, ax2)
```

Comment:
could you also add test for `ind` and `bw_method`, just using the pandas example is fine.
e.g. `kdf['a'].plot.kde(bw_method=0.3)`

Reply:
@charlesdong1991 Thanks for your comments!! I've fixed doc comment and add more tests as you have advised :)